server/@types/forms/index.d.ts

```diff
@@ -9,5 +9,14 @@ declare module 'forms' {
 
   export interface BarcodeEntryForm {
     barcode?: string
+    status?: BarcodeStatus
+  }
+
+  export enum BarcodeStatus {
+    CREATED,
+    CHECKED,
+    DUPLICATE,
+    EXPIRED,
+    RANDOM_CHECK,
```

Comment:
It would be nice if this could be imported from the Swagger docs - I'll see what I can do.

Reply:
Yes, that would be ideal. I copied it verbatim including its name and values from the API project, so if it could be part of the swagger spec and then imported as a type in the UI project that would be great.
I'll merge it as it is at the moment, and if we can get it in the swagger spec then we can refactor at that point 👍